doc/overview.rst

```diff
@@ -5,115 +5,116 @@ Overview
-hundreds of developers, which means that it is highly tested and
-productive.
-
-When using Dune, you give very little, high-level information to
-the build system, which in turn takes care of all the low-level
-details from the compilation of your libraries, executables, and
-documentation to the installation, setting up of tests, and setting up 
-development tools such as Merlin, etc.
-
-In addition to the normal features expected from an OCaml build system, 
-Dune provides a few additional ones that separate it from
-the crowd:
-
--  You never need to tell Dune the location of things such as libraries.
-   Dune will discover them automatically. In particular, this
-   means that when you want to re-organize your project, you need nothing other 
-   than to rename your directories, Dune will do the rest.
-
--  Things always work the same whether your dependencies are local or
-   installed on the system. In particular, this means that you can 
-   insert the source for a project dependency in your working
-   copy, and Dune will start using it immediately. This makes Dune a
-   great choice for multi-project development.
-
--  Cross-platform: as long as your code is portable, Dune will be
-   able to cross-compile it (note that Dune is designed internally
-   to make this easy, but the actual support is not implemented yet)
-
--  Release directly from any revision: Dune needs no setup stage. To
-   release your project, simply point to a specific tag. Of course, you can 
-   add some release steps if you'd like, but it isn't
-   necessary. 
-
-The first section below defines some terms used in 
-this manual. The second section specifies the Dune metadata
-format, and the third one describes how to use the ``dune`` command.
+Dune is a build system for OCaml (with support for Reason and Coq). It is not
+intended as a completely generic build system that's able to build any project
+in any language. On the contrary, it makes lots of choices in order to encourage
+a consistent development style.
+
+This scheme is inspired from the one used inside Jane Street and adapted to the
+opam world. It has matured over a long time and is used daily by hundreds of
+developers, which means that it is highly tested and productive.
+
+When using Dune, you give very little, high-level information to the build
+system, which in turn takes care of all the low-level details from the
+compilation of your libraries, executables, and documentation to the
+installation, setting up of tests, and setting up development tools such as
+Merlin, etc.
+
+In addition to the normal features expected from an OCaml build system, Dune
+provides a few additional ones that separate it from the crowd:
+
+-  You never need to tell Dune the location of things such as libraries. Dune
+   will discover them automatically. In particular, this means that when you
+   want to re-organize your project, you need nothing other than to rename your
```

Comment:
```suggestion
   want to reorganise your project, you need nothing other than to rename your
```
Removed hyphen and changed to British spelling (which we have in our Style Guide, but the Dune docs might be American spelling. Do you know?)

Reply:
I don't know if we take British or American English as convention. cc @rgrinberg do you know?

Reply:
I'd stay just stick what's already in use in dune. I don't have a preference, but let's keep it consistent.